src/suites/cts/validation/setVertexBuffers.spec.ts

```diff
@@ -0,0 +1,184 @@
+export const description = `
+setVertexBuffers validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+class F extends ValidationTest {
+  async init(): Promise<void> {
+    await Promise.all([super.init(), this.initGLSL()]);
```

Comment:
```suggestion
    return Promise.all([super.init(), this.initGLSL()]);
```

Reply:
This doesn't return `Promise<void>` anymore but `Promise<[void, void]>`. This is why I've used "await".

Reply:
Oh, gotcha.